core/metacling/src/TCling.cxx

```diff
@@ -626,6 +630,18 @@ extern "C" int TCling__AutoLoadCallback(const char* className)
    return ((TCling*)gCling)->AutoLoad(className);
 }
 
+extern "C" int TCling__AutoLoadLibrary(const char* Name)
+{
+   // FIXME: We're excluding some libraries from autoloading. Adding annotations to pcms from LinkDef files would fix this workaround.
+   std::vector<std::string> excludelibs = {"libRooStats.so", "libRooFitCore.so", "libRooFit.so", "libHistFactory.so"};
```

Comment:
Please be aware that any experiment trying out runtime modules will see *exactly* the same issue. I.e. this list of libraries will get you going for ROOT but the FIXME is really describing a non-optional requirement for the runtime modules feature!